lxqtapplication.h

```diff
@@ -56,14 +56,14 @@ class LXQT_API Application : public QApplication
     /*! Construct a LXQt application object.
      * \param argc standard argc as in QApplication
      * \param argv standard argv as in QApplication
-     * \param handleQuitSignals flag if signals SIGINT, SIGTERM, SIGHUP should be handled internaly (\sa quit() application)
+     * \param handleQuitSignals flag if signals SIGINT, SIGTERM, SIGHUP should be handled internally (\sa quit() application)
      */
     Application(int &argc, char **argv, bool handleQuitSignals);
     ~Application() override {}
     /*! Install UNIX signal handler for signals defined in \param signalList
      * Upon receiving of any of this signals the \sa unixSignal signal is emitted
      */
-    void listenToUnixSignals(QList<int> const & signolList);
+    void listenToUnixSignals(QList<int> const & signalList);
```

Comment:
You forgot to change the argument name also in the definition of the function `Application::listenToUnixSignals`, near the end of `lxqtapplication.cpp`. (Please don't change anything else).

Reply:
Yes, I was confused on that one, as the comment above the method declaration states `\param signalList`, yet elsewhere in `lxqtapplication.cpp` signo is used: `void listenToSignals(QList<int> const & signoList)`; inside `Application::Application(int &argc, char** argv, bool handleQuitSignals)`.

Perhaps change the comment and declaration param to `signoList`?